test/lib/testMayaUsdCreateStageInMayaRef.py

```diff
@@ -52,52 +93,23 @@ def testCreateStageInMayaRef(self):
         '''
         testDir = os.path.join(os.path.abspath('.'),'testMayaUsdCreateStageInMayaRef')
 
-        # Note: for some reason, when Maya processes file path for Maya references,
-        #       it really does not like backward slashes on Windows. So make sure
-        #       all paths use forward slashes.
-        usd_cube_scene = os.path.join(testDir, "cube.usd").replace('\\', '/')
-        maya_with_usd_ref = os.path.join(testDir, "maya_with_usd_ref.ma").replace('\\', '/')
-        maya_with_maya_ref = os.path.join(testDir, "maya_with_maya_ref.ma").replace('\\', '/')
-
-        # Create a USD scene with a cube
-        cmds.file(new=True, force=True)
-        cmds.polyCube(name="cube1")
-        USD_EXPORT_SETTING = 1
-        cmds.optionVar(iv=("mayaUsd_SerializedUsdEditsLocation", USD_EXPORT_SETTING))
-        cmds.file(usd_cube_scene, exportAll=True, force=True, type="USD Export")
-
-        # Create a Maya scene with an empty USD stage 
-        cmds.file(new=True, force=True)
-        proxy_shape = mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
-        stage = mayaUsd.lib.GetPrim(proxy_shape).GetStage()
-        self.assertTrue(stage)
-
-        # Add a reference to the usd scene with the cube
-        usdCubePath = "/usd_cube"
-        ref_prim = stage.DefinePrim(usdCubePath)
-        self.assertTrue(ref_prim)
-        ref_prim.GetReferences().AddReference(usd_cube_scene)
-
-        # Save the scene
-        cmds.optionVar(iv=("mayaUsd_SerializedUsdEditsLocation", USD_EXPORT_SETTING))
-        om.MFileIO.saveAs(maya_with_usd_ref, "mayaAscii")
-
-        # Make sure we don't retain references to USD data.
-        stage = None
-        ref_prim = None
+        usd_cube_file = createCubeUsdFile(testDir)
+        proxy_shape, usd_cube_path, maya_with_usd_ref = createMayaSceneWithStageWithUsdRef(testDir, usd_cube_file)
+        namespace = "my_ref"
+        createMayaSceneWithMayaRef(testDir, maya_with_usd_ref, namespace)
```

Comment:
Moved test code in functions, mostly so that I could be sure I was running the same code in the unit test as in the example script given in the bug ticket.